docs/3.0.0-beta.x/guides/api-token.md

```diff
@@ -0,0 +1,89 @@
+# API tokens
+
+In this guide we will see how you can create an API token system to execute request as an authenticated user.
+
+This feature is in our [backlog there](https://portal.productboard.com/strapi/1-public-roadmap/c/40-api-access-token-with-permissions).
+So this guide is about customization, to help you to have this feature. It will be no longer useful when we will release the feature.
+
+## Introduction
+
+The objective is to be able to request your API endpoints by using a query parameters `token` and be authenticated as a user. eg. `/restaurants?token=my-secret-token`.
+
+To achive this feature development, we will have to customize the `users-permissions` plugin. To do so we will use the [customization concept](../concepts/customization.md), this documentation will help you to understand how to customize all your application.
```

Comment:
achieve... This documentation will help you understand how to customize all your applications